### ISSUE TITLE
Wrong chromeandroid version parsing

### ISSUE THREAD

**Issue by nicolo-ribaudo** [browserslist/browserslist#367]
We found this bug in Babel's test suite, which a few days ago suddenly started failing.

**Current behavior:**

```js
> browserslist("chrome 53")
[ 'chrome 53' ]
> browserslist("chromeandroid 53")
[ 'and_chr 73' ]
```

**Expected behavior:**

```js
> browserslist("chromeandroid 53")
[ 'and_chr 53' ]
```

---

**UPDATE**

It looks like that the problem is that many browsers only list the last version :thinking: 

![Schermata da 2019-04-26 11-25-41](https://user-images.githubusercontent.com/7000710/56797929-1dcc2c80-6816-11e9-8227-7c539399a9b8.png)

**Comment by nicolo-ribaudo** [comment]
Ok, the problem is that caniuse only tracks the _last_ version of mobile browsers: https://caniuse.com/usage-table

I think that it would be safe to alias mobile browsers to the desktop version?

**Comment by ai** [comment]
Yeap, the problem came from Can I Use.

Technically, we can add extra option which will map mobile browsers. What mobile browsers we should also add to this map?

**Comment by nicolo-ribaudo** [comment]
I think we can at least map `and_chr` and `and_ff` to their desktop versions. As you can see in the screenshot above (https://github.com/browserslist/browserslist/issues/367#issue-437579259), there are also other mobile browsers with only one version (e.g. `and_qq`), but I don't know what they are.

**Comment by ai** [comment]
@nicolo-ribaudo do you want to send PR?

**Comment by nicolo-ribaudo** [comment]
Ok, I'll try to do it in the upcoming week

**Comment by ai** [comment]
Released in 4.6
